test_requirements.txt

```diff
@@ -0,0 +1,10 @@
+click==6.6
+flake8==2.6.2
+Flask==0.11.1
+itsdangerous==0.24
+Jinja2==2.8
+MarkupSafe==0.23
+mccabe==0.5.0
+pycodestyle==2.0.0
+pyflakes==1.2.3
+Werkzeug==0.11.10
```

Comment:
Application requirements should be in a `requirements.txt` file. This file should only have the additional requirements needed to run the tests.